config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -72,30 +28,100 @@ extensions:
         name = OpenShift Ansible Release from Local Source
         EOR
         sudo cp ./openshift-ansible-local-release.repo /etc/yum.repos.d
+        basename $(ls ${tito_tmp_dir}/noarch/ | grep atomic-openshift-utils ) .rpm >> /data/src/github.com/openshift/aos-cd-jobs/OPENSHIFT_ANSIBLE_BUILT_VERSION
```

Comment:
`basename "${tito_tmp_dir}"/noarch/atomic-openshift-utils*.rpm .rpm >> /data/src/github.com/openshift/aos--cd-jobs/OPENSHIFT_ANSIBLE_BUILT_VERSION`

have to add the adidtional `.rpm` argument canuse I need only the name without the extension